pkg/addon/addon.go

```diff
@@ -1248,6 +1290,11 @@ func (h *Installer) dispatchAddonResource(addon *InstallPackage) error {
 		return errors.Wrap(err, "render addon definitions' schema fail")
 	}
 
+	views, err := RenderViews(addon)
+	if err != nil {
+		return errors.Wrap(err, "render addon views fail")
```

Comment:
Please move this method to `dispatchAddonResource` func.

Reply:
Sorry, I think you may have accidentally read it wrong. I have put it in the `dispatchaddonresource` function.

![image](https://user-images.githubusercontent.com/30918851/173276519-6f426416-acd3-48e2-bced-a18835c9f19a.png)